sprint_statistics.rb

```diff
@@ -34,7 +34,26 @@ def default_repos
   def client
     @client ||= begin
       require 'octokit'
-      Octokit.auto_paginate = true
+      Octokit.configure do |c|
+        c.auto_paginate = true
+        c.middleware = Faraday::RackBuilder.new do |builder|
+          if ENV['DEBUG']
+            builder.use(
+              Class.new(Faraday::Response::Middleware) do
+                def on_complete(env)
+                  api_calls_remaining = env.response_headers['x-ratelimit-remaining']
+                  STDOUT.puts "DEBUG: Executed #{env.method.to_s.upcase} #{env.url} ... api calls remaining #{api_calls_remaining}"
```

Comment:
@Fryguy These improvements look great to me. My only question is why `.upcase` the `env.method.to_s` ?

Reply:
That's how it was when I stole this code from the bot.  I guess because it's common to see the method presented as GET, POST, PATCH, etc.